Packs/PaloAltoNetworks_IoT3rdParty/Integrations/PaloAltoNetworksIoT3rdParty/PaloAltoNetworksIoT3rdParty_description.md

```diff
@@ -0,0 +1 @@
+Palo Alto Networks Iot 3rd party allows users to integration Palo Alto Network IoT solution to other 3rd party products like SIEM, Cisco ISE and Service Now.
```

Comment:
If there's no detailed instructions for how to configure the integration, I think we can remove this.